composer.json

```diff
@@ -13,7 +13,8 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "scriptfusion/static-class": "^1"
```

Comment:
@Ocramius If you're not going to respond to these points and flatly refuse to use `StaticClass` you can close this PR.